docs/hello_nextflow/06_hello_config.md

```diff
@@ -259,71 +515,617 @@ NXF_CHDIR=/home/gitpod/work/34/850fe31af0eb62a0eb1643ed77b84f
+This shows the job submission details that Nextflow is trying to hand over to Slurm.
+
+!!!note
+
+    There other options that we could additionally set using other process directives to control resource allocations, which we'll get to in a little bit.
+    These would also be included in the `.command.run` file and directly passed to the Slurm execution.
 
-### 4.3. Using Executors in Profiles
+You can try using any of the other supported executors in the same way. Nextflow will translate the values submitted to the executor into the appropriate equivalent instructions.
 
-Let's combine `profiles` with `executors`. Add the following to your configuration file:
+Conveniently, you can also set up profiles to select which executor you want to use at runtime, just like we did for the Docker vs. Conda environments selection earlier.
 
-Remove the following lines:
+### 3.3. Set up profiles for executors too
+
+Let's replace the process block we had added with the executor selection profiles.
+
+_Before:_
+
+```groovy title="nextflow.config" linenums="3"
+profiles {
+    docker_on {
+        docker.enabled = true
+    }
+    conda_on {
+        conda.enabled = true
+    }
+}
 
-```groovy title="nextflow.config" linenums="17"
 process {
     executor = 'slurm'
 }
 ```
 
-Before:
+_After:_
 
-```groovy title="nextflow.config" linenums="1"
+```groovy title="nextflow.config" linenums="3"
+profiles {
+    docker_on {
+        docker.enabled = true
+    }
+    conda_on {
+        conda.enabled = true
+    }
+    local_exec {
+        process.executor = 'local'
+    }
+    slurm_exec {
+        process.executor = 'slurm'
+    }
+}
+```
+
+Although it may look like these are going to be mutually exclusive, you can actually combine multiple profiles.
+Let's try that now.
+
+### 3.4. Run with a combination of profiles
+
+To use two profiles at the same time, simply give both to the `-profile` parameter, separated by a comma.
+
+```bash
+nextflow run main.nf -profile docker_on,local_exec
+```
+
+With that, we've returned to the original configuration of using Docker containers with local execution, not that you can tell from the console output:
+
+```console title="Output"
+ N E X T F L O W   ~  version 24.02.0-edge
+
+ ┃ Launching `main.nf` [irreverent_bassi] DSL2 - revision: 66cd7c255a
+
+executor >  local (7)
+[17/82bbc4] SAMTOOLS_INDEX (2)       [100%] 3 of 3 ✔
+[8e/93609c] GATK_HAPLOTYPECALLER (2) [100%] 3 of 3 ✔
+[e6/df6740] GATK_JOINTGENOTYPING     [100%] 1 of 1 ✔
+```
+
+The point is, we can now use profiles to switch to a different software packaging system (Conda) or a different executor (such as Slurm) with a single command-line option.
+For example, if we were back on our hypothetical HPC from earlier, we would switch to using `-profile conda,slurm` in our Nextflow command line.
+
+Feel free to test that on your own to satisfy yourself that it works as expected.
+
```

Comment:
```suggestion

!!! Note
    It may be that you want a profile with settings that are different from nextflow defaults, but you still want to set it automatically without providing it with `-profile`. There's a special profile name for that called `standard`. You can see an example [in the Nextflow official docs](https://www.nextflow.io/docs/latest/config.html#config-profiles).

```